authors/apps/utils/app_util.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase, RequestFactory
```

Comment:
1 - Avoid importing what you do not need. 
2 - TestUtils would have been a much better name.
3 - I would have loved a base test class sort of implementation but we can work with this for now.